sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -373,6 +373,25 @@ object functions {
   def countDistinct(columnName: String, columnNames: String*): Column =
     countDistinct(Column(columnName), columnNames.map(Column.apply) : _*)
 
+  /**
+   * Aggregate function: returns the number of true values in a group.
+   *
+   * @group agg_funcs
+   * @since 3.0.0
+   */
+  def count_if(e: Column): Column = withAggregateFunction {
```

Comment:
I wouldn't add those functions here for now (see https://github.com/apache/spark/blob/e4f04654618b1644678c18c4331418fddb2982e8/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L42-L56)

If we're going to add, we should add it in Python and R side as well.

Reply:
> I wouldn't add those functions here for now.

Thanks for your guide. I'll remove it for now.